### ISSUE TITLE
Cloudpilot Server: Network sync broken with Parallels 18?

### ISSUE THREAD

**Issue by danbae56** [cloudpilot-emu/cloudpilot-emu#39]
Hi,

I have just upgraded my Parallels Desktop for Mac, which I use for running Palm Desktop under Windows 7, to version 18. Now my Cloudpilot Server sync setup no longer works. The proxy server seems OK, as it can be contacted using the Test button under Settings, and the Terminal then shows alerts of the type "cloudpilot-server:INFO: 2022-11-27 17:27:40,852: issued token, lifetime 10 seconds". However, nothing happens when I try to perform HotSync. No alerts from the proxy server, no nothing. The IP address in HotSync settings matches the IPv4 address given by ipconfig/all in the DOS command window. I have rebooted Cloudpilot and reset the Palm session (Tungsten W) as well as restarted Windows and Parallels. Can you give me any hints on how to proceed?

Parallels network setting: Bridged/Wifi
MacOS: 12.6

Best,
Dan

**Comment by DirtyHairy** [comment]
Is the Proxy server running on windows, or is it running on MacOS? Does network access on the pilot in work (for example using Eudora Web or Xiino)?

**Comment by danbae56** [comment]
The proxy server is running on the same Mac as Parallels. EudoraWeb connects without problems.

**Comment by DirtyHairy** [comment]
Hm, then the issue is between the proxy and Palm Desktop on the virtual machine, not between the emulator and the proxy. I am afraid that I have no good idea how to debug this (other than using wireshark to check whether a connection is established). As an alternative you could try to run the windows version of the proxy inside the virtual machine and connect to that from CP in your browser. From there you have to connect to localhost in order to hotsync.

**Comment by danbae56** [comment]
Hi again... in my initial attempts to connect, the HotSync connection mode had accidentally been switched to "Local" instead of "Modem". After correcting that, the proxy server did send messages to Terminal when I tried to sync, but there was still no reaction from Palm Desktop. 
Anyway, I finally managed to identify the problem – the network on the VM was assigned the role of "Public network" in the Windows Control Panel. After changing to "Home network" I succeeded to make the connection and perform HotSync. Now, for some reason, the HotSync progress window is hidden – I have been searching for a setting that controls this, but so far no luck. Oh well, the sync works at least, that's the important thing :)
